Doc/library/pathlib.rst

```diff
@@ -552,7 +552,7 @@ Pure paths provide the following methods and properties:
       Traceback (most recent call last):
         File "<stdin>", line 1, in <module>
         File "pathlib.py", line 694, in relative_to
-          .format(str(self), str(formatted)))
+          .format(str(self), str(formatted))
```

Comment:
It was correct. Revert this change.

Reply:
Agreed.  Matching '(' is on previous line of pathlib.py.

Reply:
```suggestion
          .format(str(self), str(formatted)))
```